include/wintls/detail/sspi_types.hpp

```diff
@@ -29,4 +29,51 @@
 #define UNICODE
 #endif // WINTLS_UNICODE_UNDEFINED
 
+#include <SubAuth.h>
```

Comment:
Please move this up between the `#ifdefs` where the two other includes are. Hopefully that doesn't cause any issues. Otherwise we should look into that.